liquibase-core/src/main/resources/www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.8.xsd

```diff
@@ -0,0 +1,1400 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@nvoxland, hi -Is it usual for us to update an older XSD version? Perhaps this is some strange merge issue?